test/test/hydraters/index.js

```diff
@@ -61,6 +61,27 @@ hydraters[env.hydraters.pdf] = {
   }
 };
 
+hydraters[env.hydraters.pdfocr] = {
+  payload: {
+    file_path: "https://raw.githubusercontent.com/AnyFetch/anyfetch-test/0eed4b59315d83c99793d54e8c7ce8391161ed51/test/test/hydraters/samples/pdfocr.anyfetch.com.test.pdf",
+    callback: "http://echo.anyfetch.com/" + uuid.v4() + '?_echo_reply=204',
+    priority: 100,
+    document: {
+      document_type: 'document',
+      metadata: {
+        path: 'pdfocr.anyfetch.com.test.pdf',
+        mime_type: 'application/pdf'
+      },
+      data: {},
+      identifier: 'pdfocr-test'
+    }
+  },
+  expected: function(data) {
```

Comment:
you can use

`expected: generateCompareFunction('./samples/pdfocr.anyfetch.com.expected.json')`

:D

(sorry ><)